static/js/stream_muting.js

```diff
@@ -67,4 +68,20 @@ export function update_is_muted(sub, value) {
         )}'] #sub_is_muted_setting .sub_setting_control`,
     );
     is_muted_checkbox.prop("checked", value);
+
+    const row = $(`.stream-row[data-stream-id='${CSS.escape(sub.stream_id)}']`);
+    disable_muted_stream_checkboxes(row, value);
+}
+
+export function disable_muted_stream_checkboxes(row, value) {
```

Comment:
Would be better to have some other name of this function because it does both disabling and enabling of checkboxes.